lldb/cmake/modules/LLDBFramework.cmake

```diff
@@ -71,6 +71,7 @@ endif()
 # At configuration time, collect headers for the framework bundle and copy them
 # into a staging directory. Later we can copy over the entire folder.
 file(GLOB public_headers ${LLDB_SOURCE_DIR}/include/lldb/API/*.h)
+file(GLOB built_public_headers ${LLDB_FRAMEWORK_ABSOLUTE_BUILD_DIR}/../tools/lldb/include/lldb/API/*.h)
```

Comment:
@bulbazord @JDevlieghere I'm pretty sure this is incorrect in a standalone build. Do you have any idea how I could properly derive the LLDB_OBJ_ROOT that contains API|Core|Target|... here?

Reply:
Yeah that is definitely incorrect for a standalone build. You could put the directory into a CMake variable and use that instead of reconstructing it here.

Also nit: `built_public_headers` -> `generated_public_headers`?